.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,22 @@
+name: Node.js Package
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
```

Comment:
isn't this the default when `npm` gets installed?

Reply:
Yeah, but the template I used for this action already had this defined so I prefered not to change it just in case :P